server/meta.jsx

```diff
@@ -5,8 +5,8 @@
 import urlLib from 'url';
 
 import { ENV, SDK_PATH, SDK_QUERY_KEYS, SDK_SETTINGS } from '@paypal/sdk-constants';
-import { node, html } from 'jsx-pragmatic';
-import { ATTRIBUTES } from 'belter';
+import { node, html } from '@krakenjs/jsx-pragmatic';
```

Comment:
This line is likely not tree shaking `jsx-pragmatic`.

Reply:
I think that's okay. Any of the files in the `server` folder are only run on the server-side with Node.js.